tests/taipy/gui/extension/test_library.py

```diff
@@ -57,7 +57,7 @@ class MyLibrary(ElementLibrary):
             inner_properties={
                 "with_property": ElementProperty(
                     PropertyType.react,
-                    f"{{<tp:prop:value>}}",
+                    "{{<tp:prop:value>}}",
```

Comment:
if you remove the f I guess you need to remove one set of curly brackets

Reply:
That is absolutely true! Thx